chart/values.yaml

```diff
@@ -0,0 +1,359 @@
+    apikeyKey: kyoo_apikeys
+
+media:
+  volumes:
+    - name: media
+      persistentVolumeClaim:
+        claimName: media
+  # mounts should always be mounted to /data
+  volumeMounts:
+    - mountPath: /data
+      name: media
+      subPath: media
+
+contentdatabase:
+  provider:
+    tmdb:
+      apikeyKey: tmdb_apikey
+      existingSecret: bigsecret
+
+autosync:
+  name: autosync
+  kyoo_autosync:
+    livenessProbe: {}
+    readinessProbe: {}
+    resources: {}
+    containerSecurityContext: {}
+    extraVolumeMounts: []
+    extraArgs: []
+    extraEnv: []
+    image:
+      repository: ~
+      tag: ~
+  replicaCount: 1
+  podLabels: {}
+  deploymentAnnotations: {}
+  podAnnotations: {}
+  imagePullSecrets: []
+  service:
+    annotations: {}
+    labels: {}
+    type: ClusterIP
+  serviceAccount:
+    create: true
+    automount: true
+    annotations: {}
+    name: ~
+  extraContainers: []
+  extraInitContainers: []
+  extraVolumes: []
+
+back:
+  name: back
+  kyoo_migrations:
+    livenessProbe: {}
+    readinessProbe: {}
+    resources: {}
+    containerSecurityContext: {}
+    extraVolumeMounts: []
+    extraArgs: []
+    extraEnv: []
+    image:
+      repository: ~
+      tag: ~
+  kyoo_back:
+    livenessProbe: {}
+    readinessProbe: {}
+    resources: {}
+    containerSecurityContext: {}
+    extraVolumeMounts: []
+    extraArgs: []
+    extraEnv: []
+    image:
+      repository: ~
+      tag: ~
+    volumeMounts:
+      - mountPath: /metadata
+        name: back-storage
+  volumes:
+    - name: back-storage
+      persistentVolumeClaim:
+        claimName: back-storage
+  replicaCount: 1
+  podLabels: {}
+  deploymentAnnotations: {}
+  podAnnotations: {}
+  imagePullSecrets: []
+  service:
+    annotations: {}
+    labels: {}
+    type: ClusterIP
+  serviceAccount:
+    create: true
+    automount: true
+    annotations: {}
+    name: ~
+  extraContainers: []
+  extraInitContainers: []
+  extraVolumes: []
+
+front:
+  name: front
+  kyoo_front:
+    livenessProbe: {}
+    readinessProbe: {}
+    resources: {}
+    containerSecurityContext: {}
+    extraVolumeMounts: []
+    extraArgs: []
+    extraEnv: []
+    image:
+      repository: ~
+      tag: ~
+  replicaCount: 1
+  podLabels: {}
+  deploymentAnnotations: {}
+  podAnnotations: {}
+  imagePullSecrets: []
+  service:
+    annotations: {}
+    labels: {}
+    type: ClusterIP
+  serviceAccount:
+    create: true
+    automount: true
+    annotations: {}
+    name: ~
+  extraContainers: []
+  extraInitContainers: []
+  extraVolumes: []
+
+matcher:
+  name: matcher
+  kyoo_matcher:
+    livenessProbe: {}
+    readinessProbe: {}
+    resources: {}
+    containerSecurityContext: {}
+    extraVolumeMounts: []
+    # workaround until dedicated image is created
+    extraArgs: 
+      - matcher
+    extraEnv: []
+    image:
+      # workaround until dedicated image is created
+      repository: ghcr.io/zoriya/kyoo_scanner
+      tag: ~
+  replicaCount: 1
```

Comment:
Just to note, ATM the matcher's cache is not distributed (only an in-memory cache). I plan to add valkey/redis to fix this.

For now, adding replicas will make more requests to content db.